server/src/http/controllers/formations/formationRegion.controller.ts

```diff
@@ -0,0 +1,65 @@
+import * as express from "express"
+import { Controller, Get, Header, Hidden, OperationId, Path, Query, Request, Response, Route, SuccessResponse, Tags } from "tsoa"
+import { IApiError } from "../../../common/utils/errorManager.js"
+import { ILbaItem } from "../../../services/lbaitem.shared.service.types.js"
+import { getFormationsParRegionQuery } from "../../../services/formation.service.js"
+import { trackApiCall } from "../../../common/utils/sendTrackingEvent.js"
+
+@Tags("FormationsParRegion")
```

Comment:
```suggestion
@Tags("Formations par région")
```
C'est l'intitulé de la section, elle peut être formaté correctement